vignettes/v06-half-life-calculation.Rmd

```diff
@@ -29,6 +29,7 @@ When automatic point selection is performed for curve stripping, the algorithm d
 All sets of points that are applicable according to the current options are selected.
 
 * Drop all BLQ values, then
+* Drop all points before the end of the last dose administration, then
```

Comment:
Thank you for catching this documentation!

Please add to this line "(if dosing information is provided)".